dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/LanguageController.java

```diff
@@ -1,6 +1,6 @@
 package de.digitalcollections.cudami.admin.controller;
 
-import de.digitalcollections.cudami.admin.business.api.service.LocaleService;
+import de.digitalcollections.cudami.admin.backend.impl.repository.LocaleRepository;
```

Comment:
Braucht es im Backend noch die `impl`/`api` Unterscheidung? Bzw konkret in diesem Fall ist das `LocaleRepository` ja ein Interface und die Implementierung wird von Feign gestellt. Vielleicht nach `api` umziehen.

Reply:
Ich werde es noch nach `api` umziehen, aber wo kommt dann die zugehörige `Config`-Klasse hin?

Reply:
würd ich auch zu `api` umziehen, bzw vielleicht geht es ja sogar das als innere klasse vom repository zu machen? Braucht ja niemand anders, würde auch die Oberfläche der API verkleinern .)